### PR TITLE
Talking about the new blog from a non-technical perspective

### PR DESCRIPTION
**Pull request by lukaseppler**

### CODE REVIEW

**Comment by lxxxvi** [comment]
@lukaseppler Would it make sense to have one or two headings to loosen up the text density?

**Comment by crackofdusk** [comment]
Hopefully the documentation proposed in https://github.com/simplificator/simplificator-blog/pull/9 helps.

**Comment by tompluess** [comment]
I like the text that discribes the evolution in episodes.

> @lukaseppler Would it make sense to have one or two headings to loosen up the text density?

Maybe you could give the every episode a subtitle?

File: content/posts/new blog.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "A new, simplified blog"
+date: 2019-12-18
```
Comment (lxxxvi) [inline content/posts/new blog.md]: @lukaseppler Please update the date before it gets merged and published.

File: content/posts/new-blog.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: "A new, simplified blog"
+date: 2019-12-18
```
Comment (lxxxvi) [inline content/posts/new-blog.md]: @lukaseppler Let's change the date...

Comment (lukaseppler) [reply content/posts/new-blog.md]: Ah nuts.

Comment (lukaseppler) [reply content/posts/new-blog.md]: Done.

**Pull request merged**